.buildkite/deployment.yml

```diff
@@ -15,6 +15,8 @@ steps:
 
   - label: ":github: Deploy Artifacts"
     command: ".buildkite/steps/ghartifacts.sh"
+    retry:
```

Comment:
how many times this will be retried before eventually failing?

Reply:
Default behaviour for automatic retries if no attempts are specified is an additional 2 retries for a total of 3 (1 initial failure and 2 additional attempts).

Reply:
:+1: